src/storage.js

```diff
@@ -1,6 +1,20 @@
 class LocalStorageStore {
   getItem(key) {
-    return JSON.parse(localStorage.getItem(key));
+    let item;
+    try {
+      item = localStorage.getItem(key);
+      return JSON.parse(item);
+    } catch (e) {
+      // old versions of the wallet would save strings without converting
```

Comment:
I don't like this piece of code to be in the PR. I also don't feel it is a good approach. Changes in the local storage format will happen more times, and I think we should find a better solution to this problem. Maybe we should version the local storage data and create migrations between version. In case of error, let the user knows that something is wrong, and they should downgrade or reset the wallet.

Reply:
Agreed about not belonging to this PR. Maybe it should be on PR #37 or even on a separate PR.

Reply:
Removed from this PR. Will implement it on #37.